internal/pkg/plugininstaller/github/workflow.go

```diff
@@ -0,0 +1,25 @@
+package github
+
+import (
+	"github.com/devstream-io/devstream/internal/pkg/plugininstaller"
+	"github.com/devstream-io/devstream/pkg/util/github"
+)
+
+func BuildWorkFlowsWrapper(workflows []*github.Workflow) plugininstaller.MutableOperation {
+	mutFunc := func(options plugininstaller.RawOptions) (plugininstaller.RawOptions, error) {
```

Comment:
Why not `return func(options plugininstaller.RawOptions) (plugininstaller.RawOptions, error) {` ?